### PR TITLE
Fixed errors

### PR DESCRIPTION
**Pull request by ghost**
I'll try to fix more as more errors are found.

### CODE REVIEW

**Comment by brokiem** [comment]
lol

File: src/ethaniccc/Mockingbird/listener/MockingbirdListener.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 use pocketmine\network\mcpe\protocol\PlayerAuthInputPacket;
 use pocketmine\network\mcpe\protocol\ProtocolInfo;
 use pocketmine\network\mcpe\protocol\SetActorMotionPacket;
-use pocketmine\network\mcpe\protocol\StartGamePacket;
+use pocketmine\network\mcpe\protocol\GamePacket;
```
Comment (JavierLeon9966) [inline src/ethaniccc/Mockingbird/listener/MockingbirdListener.php:32]: You are wrong, the packet is called `StartGamePacket`

**Review by JavierLeon9966 — CHANGES_REQUESTED** [review]

**Comment by JavierLeon9966** [comment]
Also, in protocol change 428, StartGamePacket->playerMovementType property doesn't exists anymore, so you need to replace StartGamePacket->playerMovementSettings which has the movement type

File: src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php [hunk 2]
```diff
@@ -293,11 +293,11 @@ public function process(DataPacket $packet, User $user) : void{
                         $this->handleClick($user);
                         break;
                     case InventoryTransactionPacket::TYPE_USE_ITEM:
-                        switch($packet->trData->actionType){
+                        switch($packet->actionType){
```
Comment (JavierLeon9966) [inline src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php:296]: Is `$packet->trData->getActionType()`

**Review by JavierLeon9966 — CHANGES_REQUESTED** [review]

File: src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php [hunk 3]
```diff
@@ -293,11 +293,11 @@ public function process(DataPacket $packet, User $user) : void{
...
                             case InventoryTransactionPacket::USE_ITEM_ACTION_CLICK_BLOCK:
                                 /** @var Item $inHand */
-                                $inHand = $packet->trData->itemInHand;
-                                $clickedBlockPos = new Vector3($packet->trData->x, $packet->trData->y, $packet->trData->z);
+                                $inHand = $packet->itemInHand;
```
Comment (JavierLeon9966) [inline src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php:299]: This should be
```php
$inHand = $packet->trData->getItemInHand()->getItemStack();
```

**Review by JavierLeon9966 — CHANGES_REQUESTED** [review]

File: src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php [hunk 4]
```diff
@@ -293,11 +293,11 @@ public function process(DataPacket $packet, User $user) : void{
...
+                                $clickedBlockPos = new Vector3($packet->clickPos->x, $packet->clickPos->y, $packet->clickPos->z);
```
Comment (JavierLeon9966) [inline src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php:300]: This should be
```php
$clickedBlockPos = $packet->trData->getClickPos();
```

**Review by JavierLeon9966 — CHANGES_REQUESTED** [review]

File: src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php (hunk 2 above)
Comment (ghost) [reply src/ethaniccc/Mockingbird/processing/InboundPacketProcessor.php:296]: '$packet->trData->getActionType()' is error...
IThis is what I tried to fix. But that code also throws an error.

File: src/ethaniccc/Mockingbird/listener/MockingbirdListener.php (hunk 1 above)
Comment (ghost) [reply src/ethaniccc/Mockingbird/listener/MockingbirdListener.php:32]: Okay

**Pull request closed without merging**